### PR TITLE
Add Centos / Redhat / Fedora support

### PR DESCRIPTION
**Pull request by trotter**
I've added a few changes to the customink pull request, which I've verified work with Centos 6.

### CODE REVIEW

**Comment by trotter** [comment]
bump... any chance of getting this accepted?

**Comment by miketheman** [comment]
@trotter - you seem to have added on top of #11 - and that has not been accepted yet either.

Per the discussion there, it is probably better to use the `yum_repository` from the yum cookbook rather than use templates.

Also, these are not "clean" to merge automatically, making the maintainer's job more difficult - if it can be re-requested against the current `master`, then it has a higher probability of being accepted.

**Comment by trotter** [comment]
Thanks for the feedback @miketheman. I'll update the pull request later today.

**Comment by dcrosta** [comment]
@trotter what's the purpose of moving the `mongodb_instance` into a separate recipe?

Also, please see https://github.com/customink/chef-cookbooks/pull/1, where I add to @nathenharvey's pull features similar to what you have here (I hadn't seen your pull request before I did that work, sorry!). Once that's merged, I'll probably close this pull request, but thanks for taking the time to contribute!

**Comment by trotter** [comment]
@dcrosta Awesome, glad to know you've got something similar. Also happy to know it'll be getting merged soon. As for `mongodb::standalone`, I _think_ I made that for cases where `mongodb` or `mongodb::default` were not in the recipes list, but you still wanted to install mongodb in standalone mode. If I remember correctly, this was to get it working with a cube cookbook that was including the mongodb cookbook.

**Comment by dcrosta** [comment]
@trotter Can you summarize the recent changes you added to this pull request?

Also, I've merged and closed #11, so master now supports CentOS/RHEL/Fedora/Amazon by way of a redhat-compatible init script and the 10gen yum repositories.

**Comment by trotter** [comment]
@dcrosta Whoops, some of these changes should have hit a different branch :-). My thinking post your previous comment was that I'd maintain my changes for my stuff to work until you merged in your body of work, then I'd try to just switch over to what you had and abandon my branch. I think the only one that would pertain to you is 2930996, because it corrects the running username, which differs in redhat from debian.

**Comment by jamesonjlee** [comment]
appears this was fixed at 17b96b1b closing.
